### ISSUE TITLE
sensor data not updated with closed smart life

### ISSUE THREAD

**Issue by elax46** [rospogrigio/localtuya#547]
## The problem
When the smart life application is closed, the data is not updated on the home assistant, in particular it is the energy sensor

## Environment

- Localtuya version: 3.2.2
- Last working localtuya version: 3.2.2
- Home Assistant Core version: core-2021.8.8
- [X] Are you using the Home Assistant Tuya Cloud component ? 
- [X] Are you using the Smart Life in parallel ? 

## Steps to reproduce

1. keep the smart life application in the background or completely closed on your smartphone
2. the various sensor entities do not update

## Additional information

device information extracted from developer tuya
```
{ "active_time": 1629280838,
"biz_type": 18,
"category": "wg2",
"create_time": 1629280838,
"icon": "smart/icon/ay1553162414592ZG6Kk/8eeea14694f815a903ba831c8b8078ed.jpg",
"id": "265050612462ab08275a",
"ip": "",
"lat": "",
"local_key": "",
"lon": "",
"model": "PG-01",
"name": "controllo consumi",
"online": true,
"owner_id": "1292234",
"product_id": "l5yjijwcykqo57aj",
"product_name": "Power gateway",
"status": [],
"sub": false,
"time_zone": "+01:00",
"uid": "***",
"update_time": 1629453915,
"uuid": "265050612462ab08275a"
}

```

**Comment by elax46** [comment]
I close the problem as solved #549

**Comment by elax46** [comment]
Hello everyone I realized that a new version had been released with the changes #549 I had completely forgotten the matter as I had introduced the changes, which I had tested, I noticed the availability of the update and I installed it but the sensor it does not update

**Comment by vmartinv** [comment]
> Hello everyone I realized that a new version had been released with the changes #549 I had completely forgotten the question as I had introduced the changes manually, which I had tested, I installed the update but the sensor no longer updates its data @vmartinv

Can you paste the logs? You need to configure the scan interval in the config.

**Comment by elax46** [comment]
> > Hello everyone I realized that a new version had been released with the changes #549 I had completely forgotten the question as I had introduced the changes manually, which I had tested, I installed the update but the sensor no longer updates its data @vmartinv
> 
> Can you paste the logs? You need to configure the scan interval in the config.

aaa ok i didn't know i had to configure the scan interval in the config, done now it works
